extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class ArticleVideoHooks {
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
+		$wg = F::app()->wg;
+		$articleId = $wg->Title->getArticleID();
+
+		if ( isset( $wg->videoMVPArticles[$wg->cityId][$articleId] ) ) {
+			\Wikia::addAssetsToOutput( 'article_video_scss' );
+			\Wikia::addAssetsToOutput( 'article_video_js' );
+		}
+
+		return true;
+	}
+
+	public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $out ) {
+		$wg = F::app()->wg;
+		$articleId = $wg->Title->getArticleID();
+
+		if ( isset( $wg->videoMVPArticles[$wg->cityId][$articleId] ) ) {
+			$vars['wgArticleVideoData'] = [
+				'videoId' => $wg->videoMVPArticles[$wg->cityId][$articleId]['videoId'],
+			];
+		}
+
+		return true;
+	}
+
+	public static function onSkinAfterBottomScripts( $skin, &$text ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)